templates/python/.github/workflows/python-poetry-release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Default reusable workflow
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  call-workflow-passing-data:
+    uses: bakdata/ci-templates/.github/workflows/python-poetry-release.yaml@main
+    with:
+      release-type: patch # (Required) See more values at: https://python-poetry.org/docs/cli/#version
+      ref: my-awesome-ref # (Optional) if not set the ${{ github.event.repository.default_branch }} will fill the value. In this case the changes will be pushed to my-awesome-ref
+      publish-to-test: false # (Optional) By default the packages are published to TestPyPI. In this case the packages are published to PyPI
```

Comment:
Let's set this to `true` by default to avoid accidents. Please also change the secret to `${{ secrets.TEST_PYPI_TOKEN }}`

Reply:
Done, I think.

Reply:
You also need to update the comment. Or you can remove the comments and just link to the template workflow in the CI templates repo. Perhaps that's better because then we don't need to keep it up to date it in both places.

Reply:
I am not sure if I understood correctly. I deleted almost all comments and added a single comment linking to the template. Is that what you meant?

Reply:
I was referring to the comments behind each workflow parameter, especially
> By default the packages are published to TestPyPI. In this case the packages are published to PyPI
since you changed it to `true`

Reply:
This I got, but the linking part I'm not sure about.

Reply:
yes, looks good